static/app/components/events/contexts/contextCard.tsx

```diff
@@ -113,9 +113,13 @@ export default function ContextCard({
     project,
   });
 
-  const content = contextItems.map((item, i) => (
-    <ContextCardContent key={`context-card-${i}`} meta={meta} item={item} />
-  ));
+  const content = contextItems
+    .sort((a, b) => {
+      return a.subject.localeCompare(b.subject);
+    })
```

Comment:
This will alter the [order for context](https://github.com/getsentry/sentry/blob/master/static/app/components/events/contexts/index.tsx#L40-L45) on issues details page as well. I preserved that order because it was what was there before, but I don't see a reason why we can't change it. Alphabetically seems good to me 👍

Reply:
Heyo @leeandher, it only changes the order of the items inside the card. The context ordering is one level above this, it won't change.

Reply:
Noted @leeandher, lets give sorting alphabetically a go.

Reply:
> @Abdkhan14 .sort mutates the original array, so this might in fact change the order. It would be safer to create a copy of contextItems before sorting


<img width="636" alt="Screenshot 2024-05-21 at 12 17 24 PM" src="https://github.com/getsentry/sentry/assets/60121741/f49463fd-70b7-4547-97e6-014ca966f085">